src/workerCommunication.js

```diff
@@ -14,33 +59,75 @@ function getPathPrefix() {
     return THIS_SCRIPT.src.substring(0, THIS_SCRIPT.src.lastIndexOf('/')) + '/';
 }
 
+// Destroy the worker and clear the promise so that calling initWorker will make a new one.
+function clearWorker() {
```

Comment:
This was necessary because the statefulness of the worker made testing initialization processes impossible without a way to reset the worker state.